### PR TITLE
Document `ctx.entities()`

### PR DESCRIPTION
**Pull request by roziscoding**

### CODE REVIEW

**Comment by roziscoding** [comment]
Related to https://github.com/grammyjs/grammY/pull/276

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]

**Review by KnorpelSenf — APPROVED** [review]
Please fix CI, otherwise LGTM

**Comment by roziscoding** [comment]
> Please fix CI

I was on my way to see why it broke when things caught fire lol

**Comment by roziscoding** [comment]
image
🙃

**Comment by KnorpelSenf** [comment]
That's my fault because my suggestion didn't have the right line width for the table. For once it's not GitHub that screwed up 🥲

**Comment by KnorpelSenf** [comment]
> image 🙃

Oh wait what

**Comment by roziscoding** [comment]
`deno fmt` to the rescue

**Review by KnorpelSenf — APPROVED** [review]
Thanks!

File: site/docs/guide/context.md [hunk 1]
```diff
@@ -79,6 +80,19 @@ bot.on("edited_message", (ctx) => {
   // Get the new, edited, text of the message.
   const editedText = ctx.msg.text;
 });
+
+bot.on("message:entities", (ctx) => {
+  // Get all entities present in the message text
+  const entities = ctx.entities();
+  // Gets the text for the first entity
```
Comment (roj1512) [inline site/docs/guide/context.md:87]:
```suggestion
  // Get the first entity's text
```

File: site/docs/guide/context.md [hunk 2]
```diff
@@ -79,6 +80,19 @@ bot.on("edited_message", (ctx) => {
   // Get the new, edited, text of the message.
   const editedText = ctx.msg.text;
 });
+
+bot.on("message:entities", (ctx) => {
+  // Get all entities present in the message text
```
Comment (roj1512) [inline site/docs/guide/context.md:85]:
```suggestion
  // Get all the entities.
```

File: site/docs/guide/context.md [hunk 3]
```diff
@@ -79,6 +80,19 @@ bot.on("edited_message", (ctx) => {
...
+  const entities = ctx.entities();
+  // Gets the text for the first entity
+  entities[0].text;
+
+  // Get emails present in the message text
```
Comment (roj1512) [inline site/docs/guide/context.md:90]:
```suggestion
  // Get email entities.
```

File: site/docs/guide/context.md [hunk 4]
```diff
@@ -79,6 +80,19 @@ bot.on("edited_message", (ctx) => {
...
+  const emails = ctx.entities("email");
+
+  // Get phones and emails present in the message text
```
Comment (roj1512) [inline site/docs/guide/context.md:93]:
```suggestion
  // Get phone and email entities.
```

**Review by roj1512 — APPROVED** [review]
It is unquestionable like this, thank you!

**Comment by KnorpelSenf** [comment]
This can be fully translated, but it should not be published before the 1.12 release since the code should ideally be shipped before we roll out new docs.

**Comment by fadzikri** [comment]
@CikiMomogi Where should i review that changes?

**Review by fadzikri — APPROVED** [review]
Okay, it's seems good.

**Review by habemuscode — APPROVED** [review]

**Review by roj1512 — APPROVED** [review]

**Comment by KnorpelSenf** [comment]
Great idea with the label, thanks!

**Comment by KnorpelSenf** [comment]
@roj1512 this can be merged after Chinese is removed. The respective feature has landed in the core library.

**Pull request merged**